eras/alonzo/impl/testlib/Test/Cardano/Ledger/Alonzo/Arbitrary.hs

```diff
@@ -253,23 +253,22 @@ genValidCostModel lang = do
   either (\err -> error $ "Corrupt cost model: " ++ show err) pure $
     mkCostModel lang newParamValues
 
-genValidCostModelPair :: Language -> Gen (Language, CostModel)
-genValidCostModelPair lang = (,) lang <$> genValidCostModel lang
+genValidCostModels :: Set.Set Language -> Gen CostModels
+genValidCostModels = fmap mkCostModels . sequence . Map.fromSet genValidCostModel
 
```

Comment:
This is not strictly related to this change, but since we removed one useless function (genValidCostModelPair), maybe we could also remove it's exact copy a few lines above, `genCostModel`, or at least reduce it's scope, since it's used only in [one place](https://github.com/IntersectMBO/cardano-ledger/pull/3954/commits/9d82ccdab94d04e52aa2bc7de35bc398b437624e#diff-334ba93db40dbbd431eb2831c9fe3226881e21c81945dc0dd3551ca9f215b62dL248), in the Arbitrary instance of Costmodel. 
Just because it's polluting a little the outline of the module that has become much clearer with this PR. But just an idea, it doesn't make much difference

Reply:
Great catch. Done